test/integration/backward_compatible/parallel/map/MapLockTest.js

```diff
@@ -87,44 +86,39 @@ describe('MapLockTest', function () {
         await testFactory.shutdownAll();
     });
 
-    it('should acquire the lock when key owner terminates', function (done) {
-        let clientTwo;
-        let keyOwner;
-        let key;
-
-        RC.startMember(cluster.id).then((m) => {
-            keyOwner = m;
-            return testFactory.newHazelcastClientForParallelTests({
+    it('should acquire the lock when key owner terminates', async function () {
+        let done = false;
+        const keyOwner = await RC.startMember(cluster.id);
+        const clientTwo = await testFactory.newHazelcastClientForParallelTests({
                 clusterName: cluster.id,
                 properties: {
                     ['hazelcast.client.invocation.timeout.millis']: INVOCATION_TIMEOUT_FOR_TWO
                 }
-            }, [member, m]);
-        }).then((c) => {
-            clientTwo = c;
-            return waitForNewPartitionTable(client, keyOwner);
-        }).then(() => {
-            key = generateKeyOwnedBy(client, keyOwner);
-            return map.lock(key);
+        }, [member, keyOwner]);
+        await waitForNewPartitionTable(client, keyOwner);
+        const key = generateKeyOwnedBy(client, keyOwner);
+        await map.lock(key);
+        // try to lock concurrently
+        let mapOnTwo;
```

Comment:
```suggestion
        const mapOnTwo = await clientTwo.getMap('test');
```
can be simplified a bit with